src/udev/udev-rules.c

```diff
@@ -1424,11 +1424,11 @@ static void udev_check_conflicts_duplicates(UdevRuleLine *line) {
 
                         if (new_duplicates) {
                                 duplicates = new_duplicates;
-                                log_line_warning(line, "duplicate expressions");
+                                log_line_notice(line, "style: duplicate expressions.");
```

Comment:
Yet again, this might be an indication of a real bug, so I'd suggest to keep it as a regular non-style warning.